.github/workflows/ExecuteQuery_build_and_test_on_main.yml

```diff
@@ -10,9 +10,12 @@ on:
 
 jobs:
   build:
-    uses: FrendsPlatform/FrendsTasks/.github/workflows/linux_build_main.yml@main
+    uses: FrendsPlatform/FrendsTasks/.github/workflows/build_main.yml@main
     with:
       workdir: Frends.MicrosoftSQL.ExecuteQuery
-      prebuild_command: docker-compose -f ./Frends.MicrosoftSQL.ExecuteQuery.Tests/docker-compose.yml up -d
+      env_var_name_1: FRENDS_MICROSOFTSQL_CONNSTRING
```

Comment:
I think we are using Docker container for testing - why is this needed? With docker we can assume the address of the server when testing locally. Was it somehow not working?